support/integrations/power-mta.md

```diff
@@ -11,27 +11,38 @@ Next, [create an API key](https://app.sparkpost.com/account/credentials) with th
 
 Now [create and verify](https://app.sparkpost.com/account/sending-domains) the domain you want to use for sending. The setup process should take about 5-10 minutes if you have access to your DNS server. See [Verify Sending Domains](https://www.sparkpost.com/docs/getting-started/getting-started-sparkpost/#step-2-verifying-domain-ownership) for more information.
 
-Add the following to your PowerMTA config, changing the value for auth-password to the SparkPost API key you created above:
+
+## PowerMTA basic configuration
+
+Add the following to your PowerMTA config, changing the value for auth-password to the SparkPost API key you created above.
+
+The `{sparkpost}` directive is a short way to get the needed configuration.
```

Comment:
(nit) we could refer to `{sparkpost}` as a special domain, like we refer to it in the UsersGuide, rather than a directive i.e. a directive usually refers to the name of the keyword in a key/value type of setting).

Reply:
Good catch - done.